src/solvers.jl

```diff
@@ -244,6 +245,7 @@ function solve(m::Model; suppress_warnings=false,
     # limit or tolerance is set (:UserLimit)
     if !(stat == :Infeasible || stat == :Unbounded)
         try
+            objBound = MathProgBase.getobjbound(m.internalModel) + m.obj.aff.constant
```

Comment:
We can, but it would be nice to start documenting these implicit behaviors in MathProgBase.